journal/modernized/11-chapter-10.adoc

```diff
@@ -1,10 +1,10 @@
-== Section X.
+== Chatper X
 
-Of the just judgment of God upon George Clapham, the cruel priest.
-In 1682, William Edmundson was excommunicated,
-and had a public conference with the bishop, in his court, about tithes.
-Gospel ministers, worship, etc.
-Of his voyage to the West Indies, in 1683.
+[.chapter-synopsis]
+* The just judgment of God upon George Clapham, the cruel priest
+* In 1682, William Edmundson was excommunicated, and had a public conference with the bishop, in his court, about tithes
+* Gospel ministers, worship, etc.
```

Comment:
The words _Gospel ministers, worship, etc_. should follow the word _tithes,_ on the line above, as a continued list of subjects in their public conference. (Or is this an example of where you said the editor was making its own asterisk?)

Reply:
This definitely could be an example, I really don't think I would have done that!!